mmap_span/mmap_span.go

```diff
@@ -19,11 +19,30 @@ func (ms *MMapSpan) Append(mMap mmap.MMap) {
 	ms.mMaps = append(ms.mMaps, mMap)
 }
 
+func (ms *MMapSpan) Flush() (errs []error) {
+	ms.mu.Lock()
+	defer ms.mu.Unlock()
+	for _, mMap := range ms.mMaps {
+		err := mMap.Flush()
+		if err != nil {
+			errs = append(errs, err)
+		}
+	}
+	// This is for issue 211.
+	ms.mMaps = nil
```

Comment:
This seems strange, why woudl Flush do this, especially if htere is no error?

Reply:
yes, mistake. removed.